src/components/fillProfileForm/cardInfoCard/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+import {
+  StyledContainer,
+  StyledRow,
+  StyledSmallInput,
+} from 'components/fillProfileForm/cardInfoCard/styles';
+import {
+  StyledCard,
+  StyledFormContainer,
+  StyledInput,
+  StyledSubtitle,
+  StyledTitle,
+} from 'components/fillProfileForm/sharedStyles';
+
+const expDatePH = 'MM/YYYY';
```

Comment:
it should be in common constants as it will be reused through full app

Reply:
component reworked completely and it is fixed